docs/dotnet.rst

```diff
@@ -0,0 +1,134 @@
+Dotnet
+======
+
+Prerequisites
+~~~~~~~~~~~~~
+- The application is built with a Linux-compatible .NET desktop application framework such as:
+    - `Avalonia UI <https://avaloniaui.net/>`_
+    - `Uno Platform <https://platform.uno/>`_
+    - `Eto <https://github.com/picoe/Eto>`_
+    - `GTKSharp <https://github.com/GtkSharp/GtkSharp>`_
+- The application's source code is hosted on a Git server such as GitHub, GitLab, or Bitbucket
+- A Git commit with a `tag <https://git-scm.com/book/en/v2/Git-Basics-Tagging>`_ exists on the repository to use for the build. This usually represents the version number of the application (e.g. v1.0, v1.1, etc.).
+
+Steps for Packaging
+~~~~~~~~~~~~~~~~~~~
+
+Installing dependencies
+^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
```

Reply:
I've updated this title to be a bit more descriptive, rather than removing it. This helps with the sidebar navigation a bit, since it starts on step 1.